documentation/style-guide.md

```diff
@@ -0,0 +1,249 @@
+- Use a "note" to inform the reader of points of potential confusion or to warn of potential pitfalls.
+  - A "note" occupies its own line with line breaks before and after.
+  - A "note" uses the Markdown blockquote (`>`) format followed by the word "Note:" in bold (`**`), followed by the note text:
+    - `> **Note:** Note Text.`
+  - A "note" should be brief. Generally, one line of text.
+- Be as concise as possible without sacrificing clarity.
+- Always check for grammar and spelling errors.
+
+## Headers/Sections
+
+- Do not use any `h1` headers (in Markdown preceded by `#`).
+- Do not include a title header, that is automatically provided by the metadata.
+- Do not include a header before the introductory paragraph(s).
+- Main headers should be `h2` (`##`).
+- Sub headers should be `h3` (`###`).
+- Headers should be written in Title Case.
+- Headers should not include ending punctuation.
+- Headers should not exceed 5 words.
+- Line breaks should precede and follow each header/sub header.
+
+Concept and term entries for a language function, method, command or other language structure will typically have two or three sections with the corresponding headers:
+
+- `## Syntax` for the section introducing the syntax of the term.
+- `## Example` for the section with a code block showing the term in use.
+- `## Codebyte Example` for those languages that support codebytes, the section with a runnable codebyte example.
+- There can be more headers and sub headers in cases where the complexity of the subject warrants it.
+
+## Code
+
+Markdown inline code syntax, i.e. single backticks (\`...\`), should be used for the following:
+
+- Classes, methods, functions, commands and other language keywords.
+- References, such as variable or macro names, referring to a code block.
+- Numbers and operators when referring to their use in code.
+- Expressions and inline examples of code.
+- File paths.
+
+In code blocks:
+
+- Do not use tabs for indents. Use two spaces.
+- In code examples, the entire block should be runnable as written.
+- All code blocks should be syntactically correct.
+- Examples should be complete and not be broken into smaller blocks.
+- Comments should be capitalized and punctuated like a normal sentence.
+- Line breaks should precede and follow each code block.
+- There are four general types of code blocks in Docs:
+  - "Syntax" blocks.
+    - Used with a `## Syntax` header.
+    - Shows the general syntax of a language element.
+    - Use ` ```pseudo ` blocks.
+    - They are usually followed by a description of parameters, return values etc.
+  - "Example" blocks.
+    - Used with a `## Example` header. (also `## Examples` or `## Example 2` etc.)
+    - Shows example code.
+    - Use blocks appropriate to the language used. (see: [Supported Code Blocks](#supported-code-blocks).)
```

Comment:
This formatting seems odd to me. Feels like this parenthetical should live within the sentence instead of as a separate fragment.